test/cli/harness/node.go

```diff
@@ -278,6 +279,15 @@ func (n *Node) StopDaemon() *Node {
 		_, _ = n.Daemon.Cmd.Process.Wait()
 		watch <- struct{}{}
 	}()
+
+	// os.Interrupt does not support interrupts on Windows https://github.com/golang/go/issues/46345
+	if runtime.GOOS == "windows" {
+		if n.signalAndWait(watch, syscall.SIGKILL, 5*time.Second) {
+			return n
+		}
+		log.Panicf("timed out stopping node %d with peer ID %s", n.ID, n.PeerID())
+	}
```

Comment:
Can split these harness changes out into another PR and rebase if needed. Just something I was running into running harness tests locally.

Ideally, we'd start running these tests in CI too now that they work. For another time though 😄